lib/nats.js

```diff
@@ -843,6 +865,9 @@ Client.prototype.publish = function(subject, msg, opt_reply, opt_callback) {
  */
 
 Client.prototype.subscribe = function(subject, opts, callback) {
+  if (this.closed) {
+    this.emit('error', SUB_CONN_CLOSED_ERR);
```

Comment:
Callback here is for message delivery and handling. There is not error presented to that callback, and adding one now where status callbacks expect it, at the beginning, would break existing code.